doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -29,48 +30,40 @@ Displaying
       :undocumented:
 
 
-   .. cmdv:: Print Term @qualid
-      :name: Print Term
-
-      This is a synonym of :cmd:`Print` :n:`@qualid` when :n:`@qualid`
-      denotes a global constant.
-
-   .. cmdv:: Print {? Term } @qualid\@@name
+.. cmd:: Print Term @smart_qualid {? @%{ {* @name } %} }
```

Comment:
My apologies for the mistake.  It might be worth pointing out the remaining difference, though:

```
Definition Module := 1.
Print Term Module.

Definition All := 1.
Print All.
Print Term All.

Definition Graph := tt.
Print Graph.
Print Term Graph.
```

etc.

Reply:
No worries, will do.  FWIW, I think "contrary" is much more commonly used.  I don't think I ever use "contrarily".  YMMV.

Reply:
I see they behave differently, but I don't know what to say about it.  The old text doesn't fully explain the difference is.  It says "This is a synonym of :cmd:`Print` :n:`@qualid` when :n:`@qualid` denotes a global constant."  What does it do if it's not a global constant?

Reply:
@jfehrle From what I get from Clément's comment, they behave the same, it's just that `Print Term` can serve to disambiguate at the grammar level between `Print @qualid` and another `Print` command, when `@qualid` is named `Module`, `All`, `Graph`, etc.

Reply:
I see.  It didn't occur to me that "Print Graph" and "Print All" were separate commands.

Reply:
That's the issue with having 45 such commands!